src/main/java/ua/danit/final_project/controllers/TaskController.java

```diff
@@ -46,8 +49,14 @@ public TaskController(TaskService taskService) {
   }
 
   @PostMapping
-  public Task create(@RequestBody Task task) {
-    return taskService.create(task);
+  public Task create(@RequestParam("task") String taskJson,
+                     @RequestParam(value = "file", required = false) MultipartFile file) throws IOException {
+
+    System.out.println(taskJson);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace this use of System.out or System.err by a logger. ([squid:S106](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3AS106))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=122&issues=AWYMwJKGvOWp7Yd6OrHq&open=AWYMwJKGvOWp7Yd6OrHq)

[ISSUE_KEY]: AWYMwJKGvOWp7Yd6OrHq